tgt_null.cpp

```diff
@@ -68,11 +86,22 @@ static int null_handle_io_async(struct ublksrv_queue *q, int tag)
 	return 0;
 }
 
-struct ublksrv_tgt_type  null_tgt_type = {
+static void null_report_zone(struct ublksrv_dev *dev, uint64_t offset_sectors,
+		struct blk_zone *zone_info)
+{
+	zone_info->start = offset_sectors * 10;
+	zone_info->len = 10;
+	zone_info->wp = 5;
+	zone_info->type = BLK_ZONE_TYPE_SEQWRITE_REQ;
+	zone_info->cond = BLK_ZONE_COND_IMP_OPEN;
+}
+
+struct ublksrv_tgt_type null_tgt_type = {
 	.handle_io_async = null_handle_io_async,
 	.init_tgt = null_init_tgt,
-	.type	= UBLKSRV_TGT_TYPE_NULL,
-	.name	=  "null",
+	.report_zone = null_report_zone,
+	.type = UBLKSRV_TGT_TYPE_NULL,
```

Comment:
Please add one zoned target for handling zoned, and it isn't good
to mess up loop/null, IMO.